### PR TITLE
Add forking step to README.md - Issues 5

### PR DESCRIPTION
**Pull request by Kristofunek**
Modified README.md to include a step for forking the repository as requested in issues: 5

https://github.com/pclubuiet/website/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,24 +3,31 @@ This is the official website of Programming club, UIET, Panjab University
 
 ## Quickstart ##
 
-First, clone the repository:
+1. Fork the repository (button found in top right corner of this repository):
 
-    git clone https://github.com/pclubuiet/website.git
+    This will create an exact copy of this repository and add it to your account.
```
Comment (divyam3897) [inline README.md]: and will add*

File: README.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ This is the official website of Programming club, UIET, Panjab University
 5. Run the migrations and collect the static files:
 
     `python manage.py collectstatic`
+
```
Comment (gurpreetsingh00885) [inline README.md]: Unnecessary change.

**Comment by Kristofunek** [comment]
@divyam3897 I believe that's the commits squashed together and changed to "and will"

**Comment by divyam3897** [comment]
@Kristofunek  as mentioned please remove the changes which are not asked for.

**Comment by divyam3897** [comment]
Fixed in https://github.com/pclubuiet/website/pull/30 Thank you for your PR
